client/state/domains/management/validation-schemas/reducer.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { DOMAIN_MANAGEMENT_VALIDATION_SCHEMAS_ADD } from 'state/action-types';
+
+export const validationSchemas = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case DOMAIN_MANAGEMENT_VALIDATION_SCHEMAS_ADD:
+			// action.schemas should contain schema objects keyed by tld, like:
+			// { uk: ukValidationSchema, fr: frValidationSchema, ... }
```

Comment:
this is a super-helpful comment - thanks for giving an example of possible input data!